docs/guided-remediation.md

```diff
@@ -0,0 +1,225 @@
+
+'In-place' remediation involves replacing vulnerable versions of packages in your lockfile with non-vulnerable versions, while still respecting the existing constraints for that dependency. This approach is usually less risky, but often fixes the least vulnerabilities.
+
+Selecting the "Modify lockfile in place" option will bring you to the in-place information page. From here, you can see which vulnerabilities can and cannot be resolved by this strategy. By default, every possible in-place patch will chosen to be applied. You may instead choose which subset of patches you wish to apply.
+
+![Screenshot of the interactive in-place results screen](images/guided-remediation-in-place-results.png)
+
+![Screenshot of the interactive in-place patch selection screen](images/guided-remediation-in-place-choose.png)
+
+If you wish to apply the proposed in-place patches, select the "Write" option to update your lockfile with the new dependency versions.
+
+{: .warning }
+The subcommand cannot revert the changes it makes to your lockfile. Make sure you backup your lockfile beforehand.
+
+{: .note }
+Writing these changes will not reinstall your dependencies. You'll need to run `npm ci` (or equivalent) separately.
+
+## Relock and Relax Direct Dependency Remediation
+
+Relocking recomputes your entire dependency graph based on your manifest file, taking the newest possible versions of all your required packages. This causes a large number of changes to your dependency graph, which potentially carries a larger risk of breakages.
+
+Selecting the "Relock" option will bring you to the relock information page. Here, you can see which vulnerabilities are present after relocking.
+
+In addition to relocking, it may be possible to further remove vulnerabilities by bumping the required version constraints of your direct dependencies. You may choose to apply these patches by selecting them on the relock page then choosing the "Apply pending patches" option. The dependency graph will then be recomputed and you may continue to select more options.
+
+![Screenshot of the interactive relock results screen with some relaxation patches selected](images/guided-remediation-relock-patches.png)
+
+The relaxation patches are presented in order of effectiveness, with patches that resolve the most vulnerabilities with the least amount of dependency change shown first.
+
+If you wish to apply your current relock & relaxation changes, select the "Write" option to update your manifest file with the new requirements and regenerate your lockfile (if provided).
+
+{: .warning }
+The subcommand cannot revert the changes it makes to your manifest and lockfile. Make sure you backup your files beforehand.
+
+{: .note }
+
+> The `package-lock.json` file is regenerated by first deleting the existing `package-lock.json` & `node_modules/` directory, then running `npm install --package-lock-only`. To recreate `node_modules/`, you'll need to run `npm ci` separately
```

Comment:
This bit feels a bit confusing to me. The first sentence does not make it clear that node_modules/ will be NOT be recreated along with package-lock.json. We should clarify that.

Reply:
I've changed it to be more explicit, but I'm not sure if I've made it any less confusing.